azure-pipelines.yml

```diff
@@ -591,7 +591,7 @@ jobs:
   displayName: Build Homebrew Formula
 
   dependsOn: BuildPythonWheel
-  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
+  condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'), not(contains(variables['Build.SourceBranch'], 'lts')))
```

Comment:
Similar to https://github.com/Azure/azure-cli/pull/32237#discussion_r2425389181, removing this task is simpler.